app/controllers/mailings_controller.rb

```diff
@@ -1,59 +1,17 @@
 class MailingsController < ApplicationController
-  before_action :normalize_params, only: [:create, :update]
-  before_action :set_mailings, only: :index
-  before_action :set_mailing, except: :index
 
   load_and_authorize_resource
 
-  def create
-    if @mailing.save
-      redirect_to @mailing, notice: 'Mailing was successfully created.'
-    else
-      render action: :new
-    end
-  end
-
-  def update
-    if @mailing.update_attributes(mailing_params)
-      redirect_to @mailing, notice: 'Mailing was successfully updated.'
-    else
-      render action: :edit
-    end
-  end
-
-  def destroy
-    @mailing.destroy
-    redirect_to mailings_url
-  end
-
   def index
+    @mailings = Mailing.order('id DESC').page(params[:page])
     authorize! :read, :mailing
   end
 
   def show
-    authorize! :read, :mailing
+    @mailing = Mailing.find(params[:id])
```

Comment:
Same as for `edit`, `load_and_authorize_resource` does already do this for you - you have the `@mailing` instance variable already available via some before hooks it automatically adds to the controller. So you should skip that line in order to spare the extra call to the database ✌️